index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+      crossorigin="anonymous"
```

Comment:
> - [ ] Good job using inline comment👍 I noticed that in your review, you left a comment under a line or block of code that doesn't fit the context of the comment. Please be mindful of where you leave your comments to avoid confusion and ensure that your feedback is clear and accurate.
>
>> To fix this, please go through your comments and ensure that they are placed in the appropriate location where they address the issue accurately. This will make it easier for the developer to understand the feedback and implement the necessary changes.😊

> - [ ] Please make sure to include a clear explanation or justification for any requested changes you make in your reviews. It's always important to include an explanation or a "why" for each requested change to help the developer understand the rationale behind it and make the necessary adjustments. This not only helps the developer to improve their code but also helps to build better communication and understanding between the reviewer and the developer.👍